src/view/set-password-confirm.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { SplitBackground } from '../component/background';
+import MainContent from '../component/main-content';
+import { PasswordCard } from '../component/password-entry';
+import { H1Text } from '../component/text';
+import { GlasButton } from '../component/button';
+import { color } from '../component/style';
+
+//
+// Set Password Confirm View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'flex-end',
+  },
+  title: {
+    textAlign: 'center',
+    marginBottom: 20,
+  },
+});
+
+const SetPasswordConfirmView = ({ store, wallet }) => (
```

Comment:
Might want to add a "back" button to this screen in the case the initial password is mistyped. (Potentially also the `reset-password-confirm` view.)

Reply:
Good call. Added a back button to all screens.